.config/autoconfig.php

```diff
@@ -1,38 +1,26 @@
 <?php
 
+require_once "util.php";
+
 $autoconfig_enabled = false;
 
-if (getenv('SQLITE_DATABASE')) {
+if (getConfigFromEnv('SQLITE_DATABASE')) {
     $AUTOCONFIG['dbtype'] = 'sqlite';
-    $AUTOCONFIG['dbname'] = getenv('SQLITE_DATABASE');
-    $autoconfig_enabled = true;
-} elseif (getenv('MYSQL_DATABASE_FILE') && getenv('MYSQL_USER_FILE') && getenv('MYSQL_PASSWORD_FILE') && getenv('MYSQL_HOST')) {
-    $AUTOCONFIG['dbtype'] = 'mysql';
-    $AUTOCONFIG['dbname'] = trim(file_get_contents(getenv('MYSQL_DATABASE_FILE')));
-    $AUTOCONFIG['dbuser'] = trim(file_get_contents(getenv('MYSQL_USER_FILE')));
-    $AUTOCONFIG['dbpass'] = trim(file_get_contents(getenv('MYSQL_PASSWORD_FILE')));
-    $AUTOCONFIG['dbhost'] = getenv('MYSQL_HOST');
+    $AUTOCONFIG['dbname'] = getConfigFromEnv('SQLITE_DATABASE');
     $autoconfig_enabled = true;
-} elseif (getenv('MYSQL_DATABASE') && getenv('MYSQL_USER') && getenv('MYSQL_PASSWORD') && getenv('MYSQL_HOST')) {
+} elseif (getConfigFromEnv('MYSQL_DATABASE') && getConfigFromEnv('MYSQL_USER') && getConfigFromEnv('MYSQL_PASSWORD') && getConfigFromEnv('MYSQL_HOST')) {
```

Comment:
database config from env also needs supported in the entrypoint script too. for mysql and postgres hosts, this should just be a matter of adding a line that runs them through the existing `file_env` function.